travis.sh

```diff
@@ -121,24 +121,39 @@ travis_time_start setup_rosws
 # Create workspace
 mkdir -p ~/ros/ws_$DOWNSTREAM_REPO_NAME/src
 cd ~/ros/ws_$DOWNSTREAM_REPO_NAME/src
-# When USE_DEB is true, the dependended packages that need to be built from source are downloaded based on .travis.rosinstall file.
-### Currently disabled
-###if [ "$USE_DEB" == false ]; then
-###    $ROSWS init .
-###    if [ -e $CI_SOURCE_PATH/.travis.rosinstall ]; then
-###        # install (maybe unreleased version) dependencies from source
-###        $ROSWS merge file://$CI_SOURCE_PATH/.travis.rosinstall
-###    fi
-###    if [ -e $CI_SOURCE_PATH/.travis.rosinstall.$ROS_DISTRO ]; then
-###        # install (maybe unreleased version) dependencies from source for specific ros version
-###        $ROSWS merge file://$CI_SOURCE_PATH/.travis.rosinstall.$ROS_DISTRO
-###    fi
-###    $ROSWS update
-###    $ROSWS set $DOWNSTREAM_REPO_NAME http://github.com/$TRAVIS_REPO_SLUG --git -y
-###fi
+case "$USE_DEB" in
+true) # When USE_DEB is true, the dependended packages that need to be built from source are downloaded based on .travis.rosinstall file.
```

Comment:
> As the code using this variable wasn't enabled yet and you changed the semantics of this variable (from true to false enabling the .rosinstall)

No, I haven't changed the semantics. I think that `USE_DEB=false` makes sense as an option that utilizes .rosinstall file, like so:

- `USE_DEB=false`: use source from .rosinstall.
- `USE_DEB=true`: use debian binary.